pyiceberg/transforms.py

```diff
@@ -106,6 +109,17 @@
 TRUNCATE_PARSER = ParseNumberFromBrackets(TRUNCATE)
 
 
+def _try_import(module_name: str, extras_name: str | None = None) -> types.ModuleType:
+    try:
+        return importlib.import_module(module_name)
+    except ImportError:
+        if extras_name:
+            msg = f"{module_name} needs to be installed. `pip install pyiceberg[{extras_name}]`"
```

Comment:
nit: pip expects quoted strings for extras
```
pip install "pyiceberg[pyiceberg_core]"
```


```suggestion
            msg = f"{module_name} needs to be installed. pip install \"pyiceberg[{extras_name}]\""
```